### PR TITLE
Clipboard refactor

### LINKED ISSUE #12745: Copying code from the website also copies the prompts
## Description
When copying code using the Copy from Clipboard javascript on the website, it also is copying the prompt
Go to 1) http://mxnet.incubator.apache.org/install/index.htmlplatform=Linux&language=Python&processor=GPU
2) use the copy to Clipboard option to copy text it has copied,
`$ pip install mxnet-cu92` earlier the shell prompt was not copied to clipboard

## Environment info (Required): 
Mac OS/Google Chrome Version 69.0.3497.100 (Official Build) (64-bit)

@aaronmarkham could you please take a look

### PR DESCRIPTION
**Pull request by ys2843**
## Description ##
fix #12745 . This PR fixes the many problems in `copycode.js` and refactored the code in a more extensible way.  Changes including
+ Fix prompt copied with code
+ Fix mistakenly declared global variables
+ Remove not functioning JQuery ui tooltip code
+ Extract language getter to the top that get the language of the code block relating to a copy button, this can cover any future changes in HTML structures.
+ Added an array of mapping `language: [prompt, ...]`on the top, new prompt/ language can be added to the filter

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:

### Changes ###
- [x] Updated `clipboard.js` to [latest stable version 2.0.6](https://github.com/zenorocha/clipboard.js) in minified format
- [x] Refactored `copycode.js`

## Comments ##
- Preview: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/

### CODE REVIEW

**Comment by ys2843** [comment]
@mxnet-label-bot add [Website, pr-awaiting-review]

**Comment by ys2843** [comment]
> http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/get_started?platform=linux&language=python&processor=gpu&environ=pip&

@aaronmarkham I tested it with all tutorials and installation guide. No prompt copied and the indent of the code are preserved

**Comment by ys2843** [comment]
@mxnet-bot run ci [centos-gpu]

File: docs/static_site/src/assets/js/clipboard.js [hunk 1]
```diff
@@ -1,818 +1,7 @@
...
...ialised - super() hasn't been called");return!e||"object"!=typeof e&&"function"!=typeof e?t:e}(this,(v.__proto__||Object.getPrototypeOf(v)).call(this));return n.resolveOptions(e),n.listenClick(t),n}function b(t,e){var n="data-clipboard-"+t;if(e.hasAttribute(n))return e.getAttribute(n)}e.default=m}],r.c=o,r.d=function(t,e,n){r.o(t,e)||Object.defineProperty(t,e,{enumerable:!0,get:n})},r.r=function(t){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})},r.t=function(e,t){if(1&t&&(e=r(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(r.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)r.d(n,o,function(t){return e[t]}.bind(null,o));return n},r.n=function(t){var e=t&&t.__esModule?function(){return t.default}:function(){return t};return r.d(e,"a",e),e},r.o=function(t,e){return Object.prototype.hasOwnProperty.call(t,e)},r.p="",r(r.s=6).default;function r(t){if(o[t])return o[t].exports;var e=o[t]={i:t,l:!1,exports:{}};return n[t].call(e.exports,e,e.exports,r),e.l=!0,e.exports}var n,o});
```
Comment (access2rohit) [inline docs/static_site/src/assets/js/clipboard.js:7]: Can you break this into multiple lines that are readable ?

Comment (ys2843) [reply docs/static_site/src/assets/js/clipboard.js:7]: Thanks for reviewing. This file is a JS library named `clipboard.js` and imported as `min` version on purposely to reduce file size and optimize loading time. On the top of this file, there is a readable comment marking the version and website.
>* clipboard.js v2.0.6
>* https://clipboardjs.com/
 
Here is the Github repo: https://github.com/zenorocha/clipboard.js

Comment (ys2843) [reply docs/static_site/src/assets/js/clipboard.js:7]: The `copycode.js` is the file that was mostly refactored in this PR.

**Pull request merged**